2-extra/2-water-bottle.js

```diff
@@ -19,18 +19,55 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+      if (this.volume = 100){
```

Comment:
Read the comment for what this function is supposed to do as it's more simple that expected.